src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -58,7 +58,16 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <None Include="Data\CodePageNameMappings.csv" />
+    <None Include="Data\PreferredCodePageNames.csv" />
     <None Include="project.json" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <!-- Generator for code mapping table, target to invoke is GenerateEncodingSource -->
```

Comment:
could you please add a comment here how such properties get used and how we can generate the encoding date on demand when having any data change?